src/App.tsx

```diff
@@ -212,15 +214,16 @@ const TeamRound = ({
   const playersAvailable = Math.min(11, 15 - roundPlayers.filter(player => player?.benched).length);
 
   return (
-    <div className="card">
-      <strong
-        className="card-header d-flex align-items-center"
-        onClick={() => {
-          setIsOpen(previous => !previous);
-        }}
-        style={{cursor: 'pointer'}}
-      >
-        <div>{unslugify(slug)}</div>
+    <div className="list-group-item">
```

Comment:
```suggestion
    <div className={`list-group-item ${isOpen && 'bg-light'}`}>
```

Maybe a background color on open sections can help make it even clearer what "belongs together"?
<img width="593" alt="image" src="https://github.com/stianjensen/sheplays-browser/assets/5347151/177e34ea-4849-461b-a843-53f21b082cbf">

Reply:
Yeah, I considered something like that. Although I think I'd want the open one to be highest contrast, so would need all other to be bg-light and this one white or something.

Now I pushed a small change to not have the inner borders extend all the way to the left – to distinguish them from the borders fully separating teams. I think that helps with the hierarchy, but the flip-side is that it will make it less clean to apply a different background color for just one section.

Reply:
One option is to perform color highlighting on entire teams, but I think perhaps the background will need to be made higher contrast in that case, to give enough room for a color in-between for 'unselected' teams.

Reply:
<img width=300 src=https://github.com/stianjensen/sheplays-browser/assets/1413267/4781512c-90ba-4589-9aa5-fe12e51562bc>